package.json

```diff
@@ -42,8 +42,8 @@
         "release": "npm run build && npm run webpack:prod && electron-builder",
         "pretest": "./node_modules/.bin/tslint 'src/**/*.ts*'",
         "lintfix": "./node_modules/.bin/tslint 'src/**/*.ts*' --fix",
-        "test": "cross-env TEST=true react-scripts test --env=jsdom --silent",
-        "test:ci": "cross-env TEST=true CI=true npm run test",
+        "test": "react-scripts test --env=jsdom --silent",
```

Comment:
@JordanEllis6809 - Reverted these changes - it was causing the jest plugin to break when clicking the "debug" links.

Reply:
Seriously sorry about introducing this, hopefully it was easy to find this root cause.